dvc/stage/__init__.py

```diff
@@ -147,9 +147,22 @@ def dvcfile(self, dvcfile):
 
     def __repr__(self):
         return "Stage: '{path}'".format(
+            path=self.path_in_repo if self.path else "No path"
+        )
+
+    def __str__(self):
```

Comment:
So previously str would return relpath, right? Why do we need `stage: ` here? Esp when we have `__repr__` above

Reply:
Ah, i see, because of path... Tbh, a bit confusing with repr, str and addressing...

Reply:
Another approach would be `__repr__` returning `stage: relpath:stage_name` and `__str__` returning presentational string `relpath:stage_name`.

But, my intention was to keep `__repr__` to be abspath from repo, but above approach can also be done.